src/components/StudentsTable/StudentsTable.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+
+
+const StudentsTable = ({rows, columns}) => (
```

Comment:
Т.к. компонент в данном контексте не привязан к студентам, имя можно сделать более абстрактное
```suggestion

const TableItems = ({rows, columns}) => (
```